### PR TITLE
Added some information about git and DevOps

### PR DESCRIPTION
**Pull request by sharma-shruti**

### CODE REVIEW

File: Notes.md [hunk 1]
```diff
@@ -6,4 +6,14 @@
 
 ```bash
 echo "This is interesting.!"
-```
\ No newline at end of file
+```
+
+Q) What is Git?
```
Comment (abhi18av) [inline Notes.md:11]: Please change this `Q)` to a markdown element like `#` / `-` / `*` etc.

File: Notes.md [hunk 2]
```diff
@@ -6,4 +6,14 @@
...
+- Git is a Distributed Version Control tool that is used to store different versions of a file in a remote or local repository. It is used to track changes in the source code. It allows multiple developers to work together. A VCS allows you to keep every change you make in the code repository.
+
+Q) What is the difference between git & GitHub?
+- Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them.
+
+Q) What is meant by DevOps?
+- DevOps is a software development strategy which bridges the gap between the development team and the Ops team of the company. 
+- The goal of DevOps is to change and improve the relationship by advocating better communication and collaboration between these two business units.
```
Comment (abhi18av) [inline Notes.md:19]: Hmm interesting point here!

The way I've always tought of it is that `DevOps` is the branch of software engineering which facilitates application development by software engineers, the process of software distribution, user-centric scalability as well as infrastructure management.

What do you think?

**Review by abhi18av — CHANGES_REQUESTED** [review]
Good progress @sharma-shruti !

@arushi-bhatt , would you like to add something here?

File: Notes.md (hunk 2 above)
Comment (sharma-shruti) [reply Notes.md:19]: Correct me if I am wrong, so you mean DevOps facilitates software engineers to manage all the phases of software development like Plan -> Code -> Build -> Deploy along with infrastructure management? Well, what I have understood so far is that DevOps is like a methodology that helps to bridge the gap between the Dev team and the Ops team. So they can work seamlessly and provide faster resolution of problems. As a result of this improved communication and collaboration business benefits in faster delivery and more stable operating environments.

**Comment by abhi18av** [comment]
I'm reverting this merge so that you can implement the changes within this PR itself.

For context, please read
https://github.com/sharma-shruti/My_Articles/pull/9#issuecomment-596177471

File: Notes.md (hunk 2 above)
Comment (abhi18av) [reply Notes.md:19]: > DevOps facilitates software engineers to manage all the phases of software development like Plan -> Code -> Build -> Deploy along with infrastructure management? 

Yes, I do think that you've summarized it accurately. 

> DevOps is like a methodology that helps to bridge the gap between the Dev team and the Ops team

This is true, however, in a startup setting you'll also be working with founders/ product manager to ensure things like the cost of `networking` needs to lowered for the website visitors in India even though there core business is in US etc. Of course, your collaboration would be there with the `BackEnd` team as well but these days people are doing `serverless` infrastructure which means a FrontEnd engineer ( Web or Mobile )  and a DevOps person is really enough to launch, sustain and even scale a business.

Here are some links you might find useful
https://aws.amazon.com/lambda/
https://aws.amazon.com/amplify/

And Google has another similar product
https://firebase.google.com/


In my limited experience, I've observed that two software engineering roles are very much highlighted in a startup - FrontEnd engineering and DevOps engineering. The former takes care of `what the user will see - how the business logic works`, the latter takes care of how do we get this `user interface` to the user in a `cost effective` way.

In short, @sharma-shruti , you'd find it more empowering to adapt a start-up/end-user oriented mindset rather than the traditional corporate DevOps philosophy. Companies, especially the ones with startup mindset, want people who take ownership at product level, which means people who think about improving the end-user experience. This will help you connect more with Founders and Core team.

**Comment by abhi18av** [comment]
@sharma-shruti , since this PR is now reverted. Please make all the changes in a separate PR.

I know the situation with Git would be messy because of this, but this is the learning goal :)

File: Notes.md (hunk 1 above)
Comment (abhi18av) [reply Notes.md:11]: The flow is that, you'll implement the suggested changes within this PR by adding more commits to this branch and then pushing them to remote.

After that's done, you hit `Resolve conversation` to indicate that this change request has been accomodated.

File: Notes.md (hunk 2 above)
Comment (abhi18av) [reply Notes.md:19]: @arushi-bhatt , would you like to add your thoughts here?
